corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js

```diff
@@ -180,9 +180,9 @@ hqDefine("cloudcare/js/formplayer/app", function () {
         data.onerror = function (resp) {
             var message = resp.human_readable_message || resp.exception || "Unexpected Error";
             if (resp.is_html) {
-                showHTMLError(message, $("#cloudcare-notifications"));
+                showHTMLError(message, $("#cloudcare-notifications"), resp.reportToHq);
```

Comment:
Doesn't this line, and the one below, need to pass `undefined` for `autoHideTime`?

Reply:
Functions in JS are just weird. There are two things specifically that make this OK:
1. Function arguments default to `undefined` so they don't need to be provided (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Functions#default_parameters)
2. Functions can accept any number of arguments regardless of what the function definition is.

Reply:
I'm still not quite following - `showHTMLError` has the param list `message, $el, autoHideTime, reportToHq`, so won't that interpret `resp.reportToHq` as `autoHideTime`, and `reportToHq` will always be undefined?

Reply:
🤦🏻‍♂️ yes (but only for `showHTMLError` and not `showError`): 4ca7face7ab